schemas/logs/aws/tests/cloud_trail_digest_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: CloudTrail Digest logs
```

Comment:
Two concerns on tests:

1. The test is in a different file/location than the log schema. Is the reason to split these out just to make concatenation easier? It seems like it will make life more difficult for our users when they want to write & test their own schemas.
2. This file seems very unapproachable to construct by hand, was this auto generated I'm assuming? A couple questions on the format:
  - why is there a `result` and a `results` field for each entry, even when the entries appear to only use one or the other? It seems more intuitive to just have a `results` field that may only contain a single entry
  - Why do we have to specify the log type for each test when they're all for the same log type in this file? Perhaps related to the following issue
  - Why use `---` to create multiple YAML documents?  Can we not create a single YAML document with multiple tests?
  - The `result` seems very difficult to construct without just running the tool, especially with all the whitespace characters which I'm assuming need to be exactly correct (if they don't need to be exactly correct, can we remove them?). I think it would be much better to let the result be a YAML map, which a user might be able to reasonably construct by hand if they wanted to.
  - For all other specification files, we capitalize the first letter of fields. Can we do that here as well?

I think many of our customers would find this difficult to use. A format like this is more aligned to our other detections as code configurations and I think it would be easier for our customers to make use of:

```
LogType: AWS.CloudTrailDigest
Tests:
  - Name: CloudTrail Digest logs
     Input: "<...>"
     Results:
       -
        awsAccountId: 123456789012,
        digestStartTime: 2020-04-21T12:28:23Z,
        digestEndTime: 2020-04-21T13:28:23Z,
        digestS3Bucket: cloudtrail-test-eu-west-1,
        digestS3Object: AWSLogs/123456789012/CloudTrail-Digest/00421T132823Z.json.gz,
        digestPublicKeyFingerprint: f0249abde0f55218ac45bd3750055109,
        digestSignatureAlgorithm: SHA256withRSA,
        p_any_aws_account_ids: '["123456789012"]',
        p_event_time: 2020-04-21T13:28:23Z,
        p_log_type: AWS.CloudTrailDigest
  - Name: CloudTrail Digest logs first record
     Input: "<...>"
     Results:
       -
        awsAccountId: 123456789012,
        digestStartTime: 2020-04-21T12:28:23Z,
        digestEndTime: 2020-04-21T13:28:23Z,
        digestS3Bucket: cloudtrail-test-eu-west-1,
        digestS3Object: AWSLogs/123456789012/CloudTrail-Digest/00421T132823Z.json.gz,
        digestPublicKeyFingerprint: f0249abde0f55218ac45bd3750055109,
        digestSignatureAlgorithm: SHA256withRSA,
        p_any_aws_account_ids: '["123456789012"]',
        p_event_time: 2020-04-21T13:28:23Z,
        p_log_type: AWS.CloudTrailDigestlogType: AWS.CloudTrailDigest
```

Sorry if I missed a previous PR/confluence doc that outline what this looked like, this is the first time I can recall seeing what these look like.

Reply:
These files are just dumped here from the panther repo.
The format is text-based currently to match what the `logtesting` package expects. 
We have discussed with @georgepsarakis of having `result(s)` be non-text. Haven't found the time to do that yet.
We can see if this can happen when we set up the testing for this repo.

Reply:
I see. Is it ok with you then if I open an issue in the panther repo with my suggestions?

Reply:
> I see. Is it ok with you then if I open an issue in the panther repo with my suggestions?

That sounds reasonable - @alxarch any concerns?